backend/hitas/models/apartment.py

```diff
@@ -44,29 +44,32 @@ class Apartment(ExternalHitasModel):
     stair = models.CharField(max_length=16)
 
     # 'Luovutushinta'
-    debt_free_purchase_price = models.PositiveIntegerField(default=0)
+    debt_free_purchase_price = models.PositiveIntegerField(null=True)
     # 'Ensisijaislaina'
-    primary_loan_amount = models.PositiveIntegerField(default=0)
+    primary_loan_amount = models.PositiveIntegerField(default=0, null=True)
     # 'Kauppakirjahinta'
-    purchase_price = HitasModelDecimalField(blank=True, null=True)
-    # '1. kauppakirjapvm'
+    purchase_price = HitasModelDecimalField(null=True, blank=True)
+    # 'Ensimmäinen ostopäivä'
     first_purchase_date = models.DateField(null=True, blank=True)
-    # '2. kauppakirjapvm'
+    # 'Viimeisin ostopäivä'
     second_purchase_date = models.DateField(null=True, blank=True)
```

Comment:
These are `1. kauppakirjapvm` and `2. kauppakirjapvm` in the old system. (`KAUPPVM1` and `KAUPPVM2` in the database). If this naming of the fields in the UI is wanted (and not just our own idea) then we should really change the names of this field as well, not just the comment.